mungegithub/mungers/approvers/owners.go

```diff
@@ -555,7 +557,7 @@ Associated issue requirement bypassed by: {{range $index, $approval := .ap.ListN
 
 The full list of commands accepted by this bot can be found [here](https://github.com/kubernetes/test-infra/blob/master/commands.md).
 
-<details {{if not .ap.IsApproved}}open{{end}}>
+<details {{if (and (not .ap.IsApproved) (ne 0 (len .ap.GetCCs)))}}open{{end}}>
```

Comment:
the goal is to keep the logic minimal in the template, as it's not the best place to debug code :-/